jsonschema-hyperschema.xml

```diff
@@ -883,6 +884,91 @@ GET /foo/
+                    against the schema containing the LDO).
+                </t>
+                <t>
+                    While an alternate context with a known URI is best set with the
+                    "anchor" keyword, the lack of a fragment identifier
+                    syntax for application/json means that it is usually not possible to
+                    change the context within a JSON instance.
+                </t>
+                <t>
+                    Even in "+json" media types that define JSON Pointer as a fragment identifier
+                    syntax, if the default context is nested within an array, it is not possible to
+                    obtain the index of the default context's position in that array in order
+                    to construct a pointer to another property in that same nested JSON object.
+                </t>
+
+                <figure>
+                    <preamble>
+                        For example, given this hyper-schema:
+                    </preamble>
+                    <artwork>
+<![CDATA[{
+    "type": "object",
+    "properties": {
+        "theThing": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string"}
+            }
+        }},
+        "examples": {
+            "title": "The collection of example for the thing",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": {"type": "integer"}
+                },
+                "links": [{
+                    "rel": "item"
+                    "href": "/examples/{id}",
+                    "anchorPointer": "1"
+                }]
+            }
+        }
+    }
+}]]>
+                    </artwork>
+                    <postamble>
+                        The "item" relation indicates that each array entry can be used to construct
+                        a link from the collection embedded in this instance to a resource for that
+                        specific item.  The default context of this link is the individual array
+                        element.  But the context of an "item" relation should be the collection,
+                        which is the entire "examples" array.  The Relative JSON Pointer of "1"
+                        accomplishes this as follows:
+                    </postamble>
+                </figure>
+                <figure>
+                    <preamble>
+                        Given this instance of media type application/json:
+                    </preamble>
+                    <artwork>
+<![CDATA[{
+    "theThing": {"name": "An actual thing"},
+    "examples": [{"id": 1234}, {"id": 5678}]
+}]]>
+                    </artwork>
+                    <postamble>
+                        The default context for the the link to "/example/1234" is the first
```

Comment:
"the the"

Reply:
Also, it should be `"/examples/1234"` I guess (same below).

Reply:
yup, correct on all accounts.  Should be fixed now.